### PR TITLE
LPS-114911

### PR DESCRIPTION
**Pull request by RitaSilveira**
**Jira Ticket** : 
LPS-134747
LPS-114911

Motivation:
Content review by categorization.

### CODE REVIEW

**Comment by RitaSilveira** [comment]
https://issues.liferay.com/browse/LPS-134747 , cc: @CarlosBrichete

**Comment by CarlosBrichete** [comment]
Hey @RitaSilveira, please modify the name of the test to another more descriptive like CategoriesShownInTable or something like that

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -38,6 +38,46 @@ definition {
 		}
 	}
 
+	@description = "LPS-114911 When the user selects vocabularies from the selection panel, vocabularies selected show in the table as headers and categories assigned to contents will show "
+	@priority = "5"
+	test CategoryReview {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:44]: Please modify the name of the test to another more descriptive

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**